packages/block-editor/src/components/inserter/media-tab/hooks.js

```diff
@@ -71,20 +71,23 @@ export function useMediaResults( category, query = {} ) {
 }
 
 function useInserterMediaCategories() {
-	const { inserterMediaCategories, allowedMimeTypes } = useSelect(
-		( select ) => {
-			const { getSettings } = select( blockEditorStore );
-			const {
-				__unstableInserterMediaCategories,
-				allowedMimeTypes: _allowedMimeTypes,
-			} = getSettings();
-			return {
-				inserterMediaCategories: __unstableInserterMediaCategories,
-				allowedMimeTypes: _allowedMimeTypes,
-			};
-		},
-		[]
-	);
+	const {
+		inserterMediaCategories,
+		allowedMimeTypes,
+		openverseMediaCategory,
+	} = useSelect( ( select ) => {
+		const { getSettings } = select( blockEditorStore );
+		const {
+			__unstableInserterMediaCategories,
+			allowedMimeTypes: _allowedMimeTypes,
+			openverseMediaCategory: _openverseMediaCategory,
+		} = getSettings();
+		return {
+			inserterMediaCategories: __unstableInserterMediaCategories,
+			allowedMimeTypes: _allowedMimeTypes,
+			openverseMediaCategory: _openverseMediaCategory,
+		};
```

Comment:
```suggestion
		const settings = select( blockEditorStore ).getSettings();

		return {
			inserterMediaCategories: settings.__unstableInserterMediaCategories,
			allowedMimeTypes: settings.allowedMimeTypes,
			openverseMediaCategory: settings.openverseMediaCategory,
		};
```

Nit, definitely non-blocking 😄

Where do we define `__unstableInserterMediaCategories` define settings? I don't see it documented below.

Reply:
I applied your suggestion here: https://github.com/WordPress/gutenberg/pull/47404/commits/da84b1f02c4b656eb63c158dfd3cacddcafaf49e, because I had already renamed the prop.

>Where do we define __unstableInserterMediaCategories define settings? I don't see it documented below.

In the `editor and edit-site` editor settings. It's not a default block editor setting and will be locked for 6.2.

Reply:
Can users also use this setting to restrict categories or disable `Openverse`? If yes, does it makes sense to have a separate option just for disabling `Openverse`?

I am mostly thinking out loud; no need for changes.

Reply:
The plan is to lock `__unstableInserterMediaCategories` and that would mean extenders do not have a way to filter/update that setting. So for `6.2` it's fine to have this separate explicit setting.

Reply:
That makes sense. If we end up stabilizing `__unstableInserterMediaCategories` and opening it to users, then let's revise the separate setting.